t/postprocess.t

```diff
@@ -0,0 +1,62 @@
+use Mojo::Base -strict;
+
+use Test::More;
+use File::Copy 'copy';
+use Mojo::File qw(path tempdir);
+use Mojo::JSON 'decode_json';
+use Cavil::Checkout;
+use Mojo::Util 'dumper';
+
+my $dir = path(__FILE__)->dirname->child('legal-bot');
+
+sub temp_copy {
+  my $from   = $dir->child(@_);
+  my $target = tempdir;
+  copy "$_", $target->child($_->basename) for $from->list->each;
```

Comment:
`Mojo::File` also has a [copy_to](https://metacpan.org/pod/Mojo::File#copy_to) method.

Reply:
I just copied it from checkout.t :)

Reply:
Heh, i think that code might be why i added `copy_to` upstream then. 😆